tests/locale/keymap_or_locale.pm

```diff
@@ -37,12 +38,9 @@ sub verify_default_keymap_textmode {
     }
     else {
         send_key('alt-f3');
```

Comment:
I guess the better way in general would be anyway to use os-autoinst to select the right console but not logging in. That was meant with https://progress.opensuse.org/issues/46508 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6613